docs/assets/header-bidding-process.excalidraw

```diff
@@ -0,0 +1,2215 @@
+      "fillStyle": "hachure",
+      "strokeWidth": 1,
+      "strokeStyle": "solid",
+      "roughness": 1,
+      "opacity": 100,
+      "angle": 0,
+      "x": -57.45052083333155,
+      "y": -776.0257936507907,
+      "strokeColor": "#0b7285",
+      "backgroundColor": "transparent",
+      "width": 106.74450134461509,
+      "height": 82.8290128812247,
+      "seed": 172383073,
+      "groupIds": [
+        "elCBXVXy7oiDdkUWHL6g4"
+      ],
+      "strokeSharpness": "sharp",
+      "boundElementIds": [
+        "vWw-znaEcpAUciV_8RGGP",
+        "k9iJduNZLQC6hO804lyr6"
+      ]
+    },
+    {
+      "type": "text",
+      "version": 406,
+      "versionNonce": 1151823371,
+      "isDeleted": false,
+      "id": "v4qMhb3PFvSwTGeAhIklc",
+      "fillStyle": "hachure",
+      "strokeWidth": 1,
+      "strokeStyle": "solid",
+      "roughness": 1,
+      "opacity": 100,
+      "angle": 0,
+      "x": -29.63008151927808,
+      "y": -745.6249990025283,
+      "strokeColor": "#0b7285",
+      "backgroundColor": "transparent",
+      "width": 51.10362271650817,
+      "height": 22.02742358470181,
+      "seed": 2045715777,
+      "groupIds": [
+        "elCBXVXy7oiDdkUWHL6g4"
+      ],
+      "strokeSharpness": "sharp",
+      "boundElementIds": [],
+      "fontSize": 17.62193886776144,
+      "fontFamily": 1,
+      "text": "Ozone",
+      "baseline": 15.027423584701811,
+      "textAlign": "center",
+      "verticalAlign": "middle"
+    },
+    {
+      "type": "ellipse",
+      "version": 477,
+      "versionNonce": 1314294821,
+      "isDeleted": false,
+      "id": "oNjGS0Hj2-17ujl8oI1fb",
+      "fillStyle": "hachure",
+      "strokeWidth": 1,
+      "strokeStyle": "solid",
+      "roughness": 1,
+      "opacity": 100,
+      "angle": 0,
+      "x": -35.82627776959989,
+      "y": -679.7492353445301,
+      "strokeColor": "#0b7285",
+      "backgroundColor": "transparent",
+      "width": 98.36129660981238,
+      "height": 81.12778775615612,
+      "seed": 89517007,
+      "groupIds": [
+        "elCBXVXy7oiDdkUWHL6g4"
+      ],
+      "strokeSharpness": "sharp",
+      "boundElementIds": [
+        "JP4EzOOD4u-0nCg7c2U81",
+        "tgmD9kcpoxBQALFpnzjB6",
+        "3gXRy0QHmQk5OutmSiqYb"
+      ]
+    },
+    {
+      "type": "text",
+      "version": 422,
+      "versionNonce": 820660011,
+      "isDeleted": false,
+      "id": "w-pnEvdtwA0dC7h7nU1fu",
+      "fillStyle": "hachure",
+      "strokeWidth": 1,
+      "strokeStyle": "solid",
+      "roughness": 1,
+      "opacity": 100,
+      "angle": 0,
+      "x": -9.218792785350487,
+      "y": -650.0992268699144,
+      "strokeColor": "#0b7285",
+      "backgroundColor": "transparent",
+      "width": 45,
+      "height": 23,
+      "seed": 1051540609,
+      "groupIds": [
+        "elCBXVXy7oiDdkUWHL6g4"
+      ],
+      "strokeSharpness": "sharp",
+      "boundElementIds": [],
+      "fontSize": 17.62193886776144,
+      "fontFamily": 1,
+      "text": "Xasis",
```

Comment:
This is wonderful if we can update the source directly.

Can we get the export to be generated by the CI?

```suggestion
      "text": "Xaxis",
```

Reply:
I used the gui: https://excalidraw.com/

Are those SVGs? :)

I'm not sure how we could automate without something like puppeteer etc.

Also there are additional steps like cropping required.

I'm not sure its worth the dev time for what would be a relatively stable diagram...?

Reply:
Automation would be cool though if we create more docs.